gaptrain/systems.py

```diff
@@ -74,8 +74,12 @@ def __add__(self, other):
 
         return self
 
-    def random(self, min_dist_threshold=1.7, with_intra=False, on_grid=False,
-               max_attempts=10000, **kwargs):
+    def random(self,
+               min_dist_threshold=1.7,
+               with_intra=False,
+               on_grid=False,
+               max_attempts=10000,
```

Comment:
max attempts 10,000 here but 2000 in other parts of code: active.py:  _ = system.random(min_dist_threshold=dist,
                                  max_attempts=2000)

Reply:
2000 was just because I was being impatient – I don't think it matters so much in the init configurations as the increments on the minimum distance are pretty small